packages/node_modules/@webex/react-redux-spark/src/component.js

```diff
@@ -40,11 +40,12 @@ const PLUGINS = ['authorization', 'logger', 'meetings', 'people', 'rooms'];
 const INTERNAL_PLUGINS = ['conversation', 'feature', 'flag', 'mercury', 'presence', 'search', 'team'];
 
 const storeSDKAdaptor = async (sdk, props) => {
-  const a = new WebexSDKAdapter(sdk);
+  if (props.spaceActivities.meet) {
```

Comment:
```suggestion
  if (!props.spaceActivities || props.spaceActivities.meet) {
```

Reply:
`spaceActivities` does not exist in the Recents Widget. Let's run the code in the `if` statement in that case.